ReactAndroid/src/main/java/com/facebook/react/uimanager/events/EventDispatcher.java

```diff
@@ -129,6 +130,24 @@ public void dispatchEvent(Event event) {
       // touch event dispatch will hit this codepath, and we simply queue them so that they
       // are dispatched once ReactContext creation completes and JS app is running.
     }
+
+    for (EventDispatcherListener listener : mListeners) {
+      listener.onEventDispatched(event);
```

Comment:
Hey, sorry this link doesn't work for me...

Anyways, I think the whole point of offloading events is to prevent JS thread being bloated by events that we can handle synchronously in native. If there is code that is being triggered "implicitly" we should prevent that from happening. When your intent is to offload events to native than you don't expect to be able to receive them in JS. If you still want to handle events on the JS, than you can use listener param of `Animated.event` like this:
```
onScroll={Animated.event(
   [{nativeEvent: {contentOffset: {x: this._scrollX}}}]
   {listener},          // Optional async listener
 )}
```
In which case we can forward events to JS, but I believe the default should be that we don't do that.
Another way to capture events in JS that will work "out of the box" is using animated value listener like this:
```
onScroll={Animated.event([{nativeEvent: {contentOffset: {x: this._someValue}}}])}
```
And then:
```
this._someValue.addListener(contentOffset => doSomethingWithTheOffset(contentOffset));
```